pkg/frontend/adminactions/kubeactions.go

```diff
@@ -72,6 +76,25 @@ func NewKubeActions(log *logrus.Entry, env env.Interface, oc *api.OpenShiftClust
 	}, nil
 }
 
+func (k *kubeActions) KubeGetPodLogs(ctx context.Context, namespace string, name string, containerName string, kubeInterface kubernetes.Interface) ([]byte, error) {
+	opts := corev1.PodLogOptions{Container: containerName}
+
+	if kubeInterface == nil {
+		err := api.NewCloudError(http.StatusNotFound, api.CloudErrorCodeResourceNotFound, "", "kubeclient's Clientset is empty in KubeActions")
+		k.log.Error(err)
+		return nil, err
+	}
+
+	r := kubeInterface.CoreV1().Pods(namespace).GetLogs(name, &opts)
+
+	result, err := r.Do(ctx).Raw()
```

Comment:
We need to make sure that `Raw()` doens't return us kube API server headers. We only need body with logs.

Reply:
It returns []byte which are actually the logs of the pod.